src/Artsy/Router/buildServerApp.tsx

```diff
@@ -131,6 +121,7 @@ export function buildServerApp(config: ServerRouterConfig): Promise<Resolve> {
           ServerApp,
           status,
           headTags,
+          styleTags,
```

Comment:
Pass the new style tags back up, which is one can pluck out out [from here](https://github.com/artsy/force/blob/master/src/desktop/apps/artwork2/server.tsx#L21) when using the new router setup.